spec/factories/lists.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :list do
+    title "MyString"
+user nil
```

Comment:
Consider giving the list a non-nil user by default. Changing this line to just `user` should do it, with correct indentation of course!